os/booting-on-interoute.md

```diff
@@ -1,196 +1,266 @@
 # Running CoreOS on Interoute VDC
 
-Interoute VDC is Europe's largest cloud services platform, which is powered by Apache Cloudstack.
+[Interoute Virtual Data Centre](https://cloudstore.interoute.com/what_is_vdc) is an Infrastructure-as-a-Service platform, which is integrated into Interoute's high-performance [global fibre optic network](https://cloudstore.interoute.com/networking).
 
-To run a single CoreOS node on Interoute VDC the following is assumed:
+This document is a guide to deploying a single CoreOS virtual machine on Interoute VDC. The CoreOS default configuration of SSH keypair-based authentication will be used.
 
-* You have an Interoute VDC account. You can easily [sign up for a free trial](http://cloudstore.interoute.com/main/TryInterouteVDCFREE) (no credit card required).
-* You have the  Cloudmonkey command line tool installed and configured on the computer that you are working on. Instructions on how to install and configure Cloudmonkey so that it can communicate with the VDC API can be found in the [Introduction to the VDC API](http://cloudstore.interoute.com/main/knowledge-centre/library/vdc-api-introduction-api).
-* You have installed OpenSSH client software. This is usually already installed in Linux and Mac OS. For Windows it can be downloaded at the [OpenSSH website](http://www.openssh.com/).
+Note: commands beginning with '$' are to be typed into the command line, and commands beginning '>' are to be typed into Cloudmonkey. Long lines of code have been broken up for display purposes, indicated by a continuation symbol '\', but if you copy the code make sure to remove any linebreaks. 
```

Comment:
My colleague wrote this originally with linebreaks for clarity of the command structure. My personal preference would be to remove all the linebreaks and the commands are then directly executable.